ruby_review_week_four.rb

```diff
@@ -26,12 +47,37 @@
 
 # b. on a second line, give that fish object’s name attribute the value of ‘Fred’ (assume you have access).
 
+
+
 # 12. Define a hash called ‘student’. This hash should have two keys, one called name, and the other called age, and assign a string to the name key, and an integer to the age key. 
+# student = {
+#   name: "Samaa",
+#   age: 30,
+# }
 
 # 13. Write a Ruby method that takes three arguments and returns the sum (total) of those three arguments.
 
+# def calculator(num1, num2, num3)
+#  puts num1 + num2 + num3
```

Comment:
careful here... we need to return the number out of the method

```rb
# instead of 
puts num1 + num2 + num3
# make sure you 
return num1 + num2 + num3
```